lasio/reader.py

```diff
@@ -270,27 +271,33 @@ def find_sections_in_file(file_obj):
     """Find LAS sections in a file.
 
     Returns: a list of lists *(k, first_line_no, last_line_no, line]*.
-        *k* is the position in the *file_obj* in bytes,
+        *file_pos* is the position in the *file_obj* in bytes,
         *first_line_no* is the first line number of the section (starting
         from zero), and *line* is the contents of the section title/definition
         i.e. beginning with ``~`` but stripped of beginning or ending whitespace
         or line breaks.
 
     """
-    k = 0
+    file_pos = int(file_obj.tell())
     starts = []
     ends = []
-    for i, line in enumerate(file_obj):
+    line_no = 0
+    line = file_obj.readline()
+    # for i, line in enumerate(file_obj):
```

Comment:
Note: io.TextIOWrapper 'locks' file_obj.tell() if run in a for loop but doesn't in the while statement.  That is why the change from ```for``` to ```while```.